### ISSUE TITLE
Radix Popover components' positioning inside fixed containers

### ISSUE THREAD

**Issue by GomezIvann** [dxc-technology/halstack-react#1469]
**Describe the bug**
When a component of halstack that uses the Radix Popover is inside a container with a fixed position, the popover stays in the same position even if that container is scrolled to preserve its relative position (fixed position = absolute position + scroll offset). The perfect example of this situation is the dropdown exported inside our Header component.

This may be caused by the fact that we are using the Portal version of the Popover.

**Expected behaviour**
The popover should stay in the same position as in its initial one. For example, in the previous example, the dropdown menu should stay below the dropdown trigger when the main view is scrolled.

**Screenshots**

![image](https://user-images.githubusercontent.com/44321109/218756842-c94deb9a-451a-491e-9e15-684134dc63eb.png)

**Comment by aidamag** [comment]
I have been checking the latest version (v1.0.4) of Radix Popover and looks like it doesn't have this problem. So comparing the version we have (v0.1.6) and the latest I have found that they changed the position of the Popover. Before it was as a directly child of the `body` tag and now it is a sibling of the component we want to use it with.

Before:
![image](https://user-images.githubusercontent.com/34685461/223364409-b98f01fc-ee31-4427-8c6b-3440767476cd.png)

After (the component using the Popover is a button):
![image](https://user-images.githubusercontent.com/34685461/223364312-3681769a-4808-406f-a3a6-8854a5eec572.png)

I have created an example with this problem: https://codesandbox.io/s/radix-popover-example-ntplcc

However, If we upgrade the version to the latest, we will need to change the API in the components that are using it and the Popover will not appear above all components.
